src/core/schema.service.impl.ts

```diff
@@ -70,36 +70,57 @@ const deleteFromArray = (key: string) => (data: object) => (valueToDelete: objec
 const getArray = (key: string) => (data: object) => {
   return data[key];
 };
-const addReference = (schema: JsonSchema, variable: string, pathToContainment: string) =>
+const addReference = (schema: JsonSchema, identifyingProperty: string, variable: string) =>
```

Comment:
`variable` is a very generic name. Can we rename this, e.g. to `prop`, `propName` or simlar?